examples/reactions.js

```diff
@@ -8,7 +8,9 @@ const config = {
 
 const params = {
   message_id: 1,
-  emoji: 'smile',
+  emoji_name: 'musical_note',
+  emoji_code: '1f3b5',
```

Comment:
I agree that it's less user-friendly.  For context, the API change here is required to properly support the possibility that e.g. the user is clicking on a deactivated realm emoji (or an emoji whose name has changed in a newer version of the emoji set), which can come up for full clients like the web and mobile apps.  Probably not a real issue for a little script using this, though.

So I think it's possible we could make the API backend support being more sloppy with these things and just do the most natural guess, since random little scripts wouldn't care about that category of error anyway.

@HarshitOnGitHub FYI.